spark/src/main/kotlin/com/adevinta/spark/components/tags/TagOutlined.kt

```diff
@@ -104,16 +179,8 @@ internal fun TagOutlinedPreview(
 ) {
     val (theme, userType) = param
     PreviewTheme(theme, userType) {
-        TagPromote {
-            Text(text = "Sponsorisé")
-        }
-
-        TagUrgent {
-            Text(text = "Nouveau")
-        }
-
-        TagPro {
-            Text(text = "Pro")
+        TagIntent.values().forEach {
+            TagOutlined("Tag information", intent = it)
```

Comment:
We should add the corresponding intent name in the previews.

```suggestion
            TagOutlined("Tag ${it.name}", intent = it)
```

Reply:
agreed